tools/tests/test_lint.py

```
@@ -188,20 +188,64 @@ def test_ledger_row_missing_field_is_a_finding(tmp_path):
+
+def test_valid_ledger_row_is_clean(tmp_path):
+    _write_ledger(tmp_path, _ledger_row())
     assert lint.run(tmp_path) == []
 
 
+def test_ledger_row_without_found_by_is_a_finding(tmp_path):
+    row = _ledger_row()
+    del row["found_by"]
+    _write_ledger(tmp_path, row)
+    findings = lint.run(tmp_path)
+    assert len(findings) == 1 and "found_by" in findings[0]
```

Ruff (0.16.1)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)